src/Components/Components/src/Sections/SectionContent.cs

```diff
@@ -12,11 +12,17 @@ internal class SectionContent : ISectionContentProvider, IComponent, IDisposable
         private SectionRegistry _registry = default!;
 
         /// <summary>
-        /// Gets or sets the name that determines which <see cref="SectionOutlet"/> instances will render
+        /// Gets or sets the name that determines which <see cref="SectionOutlet"/> instance will render
         /// the content of this instance.
         /// </summary>
         [Parameter] public string Name { get; set; } = default!;
 
+        /// <summary>
+        /// Gets or sets whether this component should provide the default content for the target
```

Comment:
Not that I mind this change - but could we this is always true (given that there's only one thing that provides it today)?

Reply:
`SectionContent` is also used in `PageTitle`: https://github.com/dotnet/aspnetcore/blob/cfc5aff291178c05f297ea7169aad35340a7a5f8/src/Components/Web/src/Head/PageTitle.cs#L23

It doesn't specify `IsDefaultContent` because it's `false` for every case other than the default title, which is rendered by `HeadOutlet`.

Am I understanding the suggestion correctly?